src/App.jsx

```diff
@@ -1,14 +1,25 @@
-import BookDetail from "./components/BookDetail/BookDetail"
 
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Nav from "./components/Nav/Nav";
+import HomePage from './components/HomePage/HomePage'
+import './App.css'
+import BookDetail from "./components/BookDetail/BookDetail";
 
 function App() {
-  
-
   return (
-    <>
-      <BookDetail />
-    </>
-  )
+    <div>
+      <BrowserRouter>
+        <Nav />
+
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+
```

Comment:
remove these spaces and move <routes> to the same line as <nav> as Geoff said

Reply:
White space removed (and new route in place):

 <Route path="book/:id" element={<BookDetail />} />